app/controllers/prospects_controller.rb

```diff
@@ -0,0 +1,65 @@
+class ProspectsController < ApplicationController
+  skip_before_action :authenticate_user!
```

Comment:
This is not true. THIS VERSION of the application will mostly be deployed as a closed product and user invites and administration will be managed similar to an enterprise model. Users may be able to request access from a central admin but otherwise the admin will be sending out invites to users individually and in groups. Please mail me for further clarification. Dont forget this is a FIND version of the application and we will get what we want and not what has previously been anticipated.